app/views/volunteer_applications/new.html.slim

```diff
@@ -0,0 +1,8 @@
+.row
+  .col-xs-12
+    h1= t('volunteer_application')
+.row
+  .col-xs-12
+    = simple_form_for(@volunteer, url: volunteer_applications_path) do |f|
+      = render 'volunteers/form', f: f
+      = f.button :submit, t('submit_application'), class: 'btn btn-success'
```

Comment:
Present your idea about sorting of translations to the team and make a decision about inclusion